### DISCUSSION TITLE
Support ingesting data from Xi Xing's MATLAB Automated Peak Picking software

### DISCUSSION THREAD

**Discussion by lparsons** [Princeton-LSI-ResearchComputing/tracebase#1056] [Ideas]
@xxing9703 is working on software that automates peak picking and outputs data in a format similar to MAVEN/Accucor files. We would like to ensure that TraceBase can ingest these files. To do so might involve some changes to TraceBase and/or the MATLAB software.

1. Get copies of output files from the MATLAB Automated Peak Picking software. @xxing9703, can you upload some example files?
2. Determine how much effort it would be support it, identify any changes we might want to make it easier.
3. Make any changes needed to the MATLAB code and repeat until we are satisfied
4. Create an issue to define support for the MATLAB output in TraceBase

**Comment by xxing9703** [comment]
[Michael_arg.xlsx](https://github.com/user-attachments/files/16289085/Michael_arg.xlsx)
[Michael_Cys.xlsx](https://github.com/user-attachments/files/16289086/Michael_Cys.xlsx)

**Comment by lparsons** [comment]
Partial list of steps to add support (in progress):
* [ ] Add data format for this software
* [ ] [`AccuCorDataLoader.detect_data_format](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/780cab25fa2d110682e634d880c4cd2a58d7a3de/DataRepo/loaders/accucor_data_loader.py#L2097)
* [ ] [Validation specific file type determination](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/780cab25fa2d110682e634d880c4cd2a58d7a3de/DataRepo/views/upload/validation.py#L201-L208)
